report/templates/benchmark.json

```diff
@@ -0,0 +1,22 @@
+{
+    "samples": [
+{% for sample in samples %}
+    {
+        "benchmark": "{{ benchmark }}",
+        "sample": "{{ sample.id }}",
+        "status": "{{ sample.status }}",
+        "compiles": "{{ sample.result.compiles }}",
+        "crashes": "{{ sample.result.crashes }}",
+        "coverage": "{{ sample.result.coverage | percent }}",
+        "coverage_diff": "{{ sample.result.line_coverage_diff }}",
+        "coverage_report": "{{ sample.result.coverage_report_path | cov_report_link }}",
+        "stacktrace": "{{ sample.stacktrace }}",
+        "target_binary": "{{ sample.target_binary }}",
+        "reproducer": "{{ sample.reproducer }}",
+        "run_log": "{{ sample.run_log }}",
```

Comment:
What's the difference between stacktrace and run_log? Would it make sense to only have run_log?

Reply:
`run_log` is the fuzzing runtime log from `libFuzzer`, which we already store.
`stackstrace` is the output from `./target_binary reproducer_input`. I added it to simplify future automatic error categorization because of its cleanliness (i.e., no fuzzer output, which can be long but useless for long fuzzing timeout). It may also help to automatically identify if the crash is reproducible.

Reply:
https://github.com/google/oss-fuzz/pull/11700/files#diff-ed7b0455ec9bfd97499697c6be191669e6f378243e24ba14981b4b7f092ffd6cR158

Reply:
Missed that this was added in https://github.com/google/oss-fuzz/pull/11700.